kategory/src/test/kotlin/kategory/data/Function1Test.kt

```diff
@@ -0,0 +1,25 @@
+package kategory
+
+import io.kotlintest.KTestJUnitRunner
+import io.kotlintest.matchers.shouldBe
+import org.junit.runner.RunWith
+
+@RunWith(KTestJUnitRunner::class)
+class Function1Test : UnitSpec() {
+    init {
+        testLaws(MonadLaws.laws(Function1.monad<Int>(), object : Eq<HK<Function1.F, Int>> {
```

Comment:
Also just realized we are missing `Category` and `MonoidK` as typeclasses, also addressable in a separate PR. `Function1` should also have an instance for `MonoidK`